Apps/Inc/Tasks.h

```diff
@@ -112,11 +112,6 @@ extern CPU_STK BLE_Stk[TASK_BLE_STACK_SIZE];
 extern CPU_STK Idle_Stk[TASK_IDLE_STACK_SIZE];
 extern CPU_STK Init_Stk[TASK_INIT_STACK_SIZE];
 
-/**
- * @brief   Queue for pushing and popping CAN Messages
- */
-OS_Q CANBus_MsgQ;
```

Comment:
This is used on a different branch with an open pull request into rtos, so I don't want to delete it